src/DataProtection/AzureStorage/src/Microsoft.AspNetCore.DataProtection.AzureStorage.csproj

```diff
@@ -11,7 +11,12 @@
 
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore.DataProtection" />
-    <Reference Include="WindowsAzure.Storage" />
+    <Reference Include="Microsoft.Azure.Storage.Blob" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(AspNetCoreMajorMinorVersion)' == '3.0'">
+    <!-- This dependency was replaced by Microsoft.Azure.Storage.Blob between 3.0 and 2.2. This suppression can be removed after 3.0 is complete. -->
```

Comment:
Need to file a bug to remember this?

Reply:
Opened https://github.com/aspnet/AspNetCore/issues/9473 and put it in the 3.1 milestone. It's not super urgent because I've written this code to only apply when building 3.0. But yes, can be deleted later when it effectively becomes dead code.